### PR TITLE
Update: In-app Pricing Page FAQs

### PR DESCRIPTION
**Pull request by dericksozo**
Close: [#63](https://github.com/orgs/amplication/projects/5?pane=issue&itemId=45409203)

### Summary
🆕💡🖋️

Updated the FAQ component to match the new pricing and features. Rewrote and added questions and answers to improve customer understanding and engagement. The change was part of a website content and design overhaul.

> _FAQ component_
> _`Amplication` changes_
> _Autumn of renewal_

### Walkthrough
*  Rewrite and add questions and answers in the FAQ component to reflect the new pricing model and features of Amplication ([link](https://github.com/amplication/amplication/pull/7522/files?diff=unified&w=0#diff-d1632ef604f51b2b472c1c0d4ab70d88de050c5115abef99f064234f3f9831a4L22-R133))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 1]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
-              workspace settings. For now, you can reach out directly to us{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Update workspace plan"
-                body="here"
-              />{" "}
-              and we will update your plan as requested.
+              Amplication offers a structured way to add custom code and modify
+              your generated code, but it still adheres to its core architecture
+              and coding patterns.
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:46]: Not sure about the sentence of "but it still adheres to its core architecture and coding patterns.".
I would talk about the custom code, and mention that our generated code meets the best practices and the highest quality.

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 2]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
-          question="Do I get a notification if I am approaching my usage limits?"
-          answer={`We constantly show you your plan limitations within your workspace. We will not block you from configuring your services but in order to generate the code you will need to comply with the workspace plan limits.
-
-          We will notify you via email or other means when we think it might be a good time for you to consider upgrading your workspace plan.`}
-        />
-        <Question
-          question="What if I need to add more team members?"
-          answer="No problem. Select the plan that meets your needs. The Free edition supports up to two user seats, the Pro edition supports up to five user seats, and the Enterprise edition has no limit. "
+          question="We have legacy systems, and would like to progress to modern architecture. Can you assist with it?"
+          answer={
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:61]: I like this answer, but I think it's too long. Maybe we can just mention the feature names, and link it to its documentation. WDYT?

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 3]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
-              />{" "}
-              for more information and to purchase.
+              Amplication allows you to deploy to AWS's Elastic Container
+              Service (ECS). We provide in-depth documentation that will walk
+              you through using various Amplication plugins to provision the
+              required infrastructure and set up a process for continuous
+              integration and deployment.
+              <br />
+              <br />
+              The steps cover manual configurations for permissions and
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:99]: Not sure about this sentence (due to the length of the answer). Again- I think we can link to the documentation page

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 4]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
+                  customizations without being bound to rigid system
+                  constraints.
+                </li>
+              </ul>
+            </div>
+          }
         />
         <Question
-          question="Can I pay for my subscription with an invoice?"
+          question="Can you assist with the actual deployment (terraform-like) in a cloud provider?"
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:89]: Is it a good question- English and technical- wise?

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 5]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
+              The steps cover manual configurations for permissions and
+              authentication, plugin settings for AWS services, code generation,
+              provisioning with Terraform, and deployment using GitHub Actions.
             </div>
           }
         />
         <Question
-          question="We're an open-source project, is there special pricing available?"
+          question="I have an existing DB with a lot of data, entities, tables, and relationships. Can I use your backend development platform, while still keeping the data and the knowledge?"
+          answer="Amplication's DB Schema Import feature provides a clear pathway for integrating an existing database with the Amplication platform. The import feature is available on all Amplication plans, including Free accounts."
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:107]: The import is available only for the Enterprise plan...

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 6]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
             <div>
-              Yes, as part of our commitment to support open-source projects, we
-              offer the pro plan for free. Reach out{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Pro plan for Open Source project"
-                body="here"
-              />{" "}
-              and we will set you up.
+              Amplication's Smart Git Sync feature ensures that changes made in
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:113]: I think the answer should be much more clear and short- "Yes, Amplication's Smart Git Sync feature ensures...", we always prefer your custom code over the generated code, and know how to combine the two automatically.
Something like that. A link to the documentation should be considered.

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 7]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
+              changes from Amplication.
             </div>
           }
         />
+        <Question
+          question="Due to security regulations, my enterprise require supporting relevant capabilities, like 2FA, audit log, SSO and more. Do you support it?"
+          answer="The Enterprise edition gives you access to advanced security features like SSO and audit logs are provided."
+        />
+        <Question
+          question="Can I pay to Amplication through my AWS account (AWS billing)?"
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:131]: I would put this question after the "What are the differences between the free and the enterprise tier?" question

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 8]
```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
...
+          answer=""
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:132]: The answer should be something like- Yes, please contact us to handle it (or such).

File: packages/amplication-client/src/Purchase/FAQ.tsx (hunk 7 above)
Comment (PazYanoverr) [reply packages/amplication-client/src/Purchase/FAQ.tsx:131]: In addition- please update the English of the question if required :)

**Review by PazYanoverr — COMMENTED** [review]
General comment-
I suggested in a few places to add links to the documentation. Not sure if it's the best practice for such FAQs - your call.

**Comment by dericksozo** [comment]
Hey @PazYanoverr, [this commit](https://github.com/amplication/amplication/pull/7522/commits/5af90df20f78f0819a61be6ff7135329f23b788c) implements all of your feedback.

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 9]
```diff
@@ -62,74 +122,72 @@ export const FAQ = () => {
...
-              an Amplication project are automatically tracked and committed to
-              a dedicated branch in the connected Git repository. The system is
-              designed to manage the syncing process efficiently, automatically
-              creating commits and pull requests with clear messages that
-              reflect the changes made.
-              <br />
-              <br />
-              Your changes will not get overriden until you have merged any new
-              changes from Amplication.
+              Yes, Amplication's Smart Git Sync feature ensures that your custom
```
Comment (PazYanoverr) [inline packages/amplication-client/src/Purchase/FAQ.tsx:174]: Oops- the answer to the question should be No, we respect your code changes :)

File: packages/amplication-client/src/Purchase/FAQ.tsx (hunk 7 above)
Comment (PazYanoverr) [reply packages/amplication-client/src/Purchase/FAQ.tsx:189]: I think these comments were not applied. Let's move the "AWS" question to the suggested location

**Comment by PazYanoverr** [comment]
@dericksozo - Added a few more comments

**Comment by dericksozo** [comment]
Great, I just implemented your feedback @PazYanoverr.

**Review by PazYanoverr — APPROVED** [review]

File: packages/amplication-client/src/Purchase/FAQ.tsx (hunk 5 above)
Comment (mulygottlieb) [reply packages/amplication-client/src/Purchase/FAQ.tsx]: @PazYanoverr , right now the import is available on all plans actually. Do you plan to change this?
The rationale was that people would be able to import but if they are limited in number of entities that would block them. Of course now when we remove the limitation on #entities, we can reconsider. Anyway this is now part of the free plan as well.

Comment (PazYanoverr) [reply packages/amplication-client/src/Purchase/FAQ.tsx]: Yes, it appears in the spec- the import Prisma schema should be available in the Enterprise plan only.
The idea is to allow importing as part of the trial phase (if required by the user). After that- it won't be available in the free plan, as it's an advanced feature.

**Review by PazYanoverr — APPROVED** [review]

**Comment by PazYanoverr** [comment]
Waiting with the merge to the reverse trial and new packaging to be implemented

**Comment by mulygottlieb** [comment]
@dericksozo please fix all the GH warnings about links with "_blank":

_Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations_

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**